test/performance/odp_l2fwd.c

```diff
@@ -1247,6 +1517,14 @@ static void usage(char *progname)
 	       "                          1: Create a pool per interface\n"
 	       "  -n, --num_pkt <num>     Number of packets per pool. Default is 16k or\n"
 	       "                          the maximum capability. Use 0 for the default.\n"
+	       "  -u, --vector_mode       Enable vectors mode.\n"
+	       "  -q, --vec_pool_per_if   Number of vector pool per interface.\n"
```

Comment:
Do we need a new parameter for this or could we use the existing `-y, --pool_per_if` option? The documentation for `-y` could be extended to state that it covers both pool types.

Reply:
Will use existing option.